src/app/kvm/components/RamResources/RamResources.test.tsx

```diff
@@ -66,12 +42,12 @@ describe("RamResources", () => {
       />
     );
 
-    expect(wrapper.find("[data-testid='page-size']").exists()).toBe(true);
-    expect(wrapper.find("[data-testid='page-size']").text()).toBe("(Size: 5B)");
+    expect(screen.getByTestId("page-size")).toBeInTheDocument();
```

Comment:
The assertion below this will fail if it isn't in the document, you shouldn't need to check it twice.

```suggestion
```